src/dev/src/app/datagrid/string-filtering/string-filtering.html

```diff
@@ -18,9 +18,9 @@ <h2>Built-in filters</h2>
 <h4>String filter</h4>
```

Comment:
Would you mind doing the same file renames like you did on the website from `string-filtering` to `built-in-filters`?